pivottablejs.py

```diff
@@ -8,22 +8,52 @@
 template = """
 <!DOCTYPE html>
 <html>
-    <head>
+    <head id="head-pivottable">
         <title>PivotTable.js</title>
 
         <!-- external libs from cdnjs -->
-        <link rel="stylesheet" type="text/css" href="%(static)s/pivot-ajax/c3.min.css">
-        <script type="text/javascript" src="%(static)s/pivot-ajax/jquery.min.js"></script>
-        <script type="text/javascript" src="%(static)s/pivot-ajax/jquery-ui.min.js"></script>
-        <script type="text/javascript" src="%(static)s/pivot-ajax/d3.min.js"></script>
-        <script type="text/javascript" src="%(static)s/pivot-ajax/jquery.csv-0.71.min.js"></script>
-        <script type="text/javascript" src="%(static)s/pivot-ajax/c3.min.js"></script>
+        <script type="text/javascript">
+        var static_url = window.location.href.split('/');
+        static_url = static_url.slice(0,6);  // keep up to app 'notebook'
+        var staticjs = static_url.join('/');
+
+        var head = document.getElementById('head-pivottable'); // append the scripts to the iframe head
+        css_scripts = ["pivot-ajax/c3.min.css", "pivottable/pivot.min.css"];
+        for (i = 0; i < css_scripts.length; i++) {
+            var link = document.createElement('link');
+            link.setAttribute('type', 'text/css');
+            link.setAttribute('rel', 'stylesheet');
+            var ref = staticjs + '/nbextensions/' + css_scripts[i];
+            link.setAttribute('href', ref);
+            head.appendChild(link);
+        }
+        js_scripts = [
+                "pivot-ajax/jquery.min.js",    // load jquery first
+                "pivot-ajax/jquery-ui.min.js",
+                "pivot-ajax/jquery.csv-0.71.min.js",
+                "pivot-ajax/d3.min.js",
+                "pivot-ajax/c3.min.js",
+                "pivottable/pivot.min.js",
+                "pivottable/d3_renderers.min.js",
+                "pivottable/c3_renderers.min.js",
+                "pivottable/export_renderers.min.js"]
 
-        <link rel="stylesheet" type="text/css" href="%(static)s/pivottable/pivot.min.css">
-        <script type="text/javascript" src="%(static)s/pivottable/pivot.min.js"></script>
-        <script type="text/javascript" src="%(static)s/pivottable/d3_renderers.min.js"></script>
-        <script type="text/javascript" src="%(static)s/pivottable/c3_renderers.min.js"></script>
-        <script type="text/javascript" src="%(static)s/pivottable/export_renderers.min.js"></script>
+        function loadScript() {
+            if (js_scripts.length == 0) {
```

Comment:
A comment here about you are going to shift the js_script array would be nice for the reader...